plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -33,6 +33,10 @@ wp-env run tests-cli wp user create customer customer@woocommercecoree2etestsuit
 	--last_name='Smith' \
 	--user_registered='2022-01-01 12:23:45'
 
+wp-env run tests-cli wp post create --post_type=page --post_status=publish --post_title='Confirmation email' --post_name='confirmation-email' --post_content='[bis_confirmation_received_email]'
+wp-env run tests-cli wp post create --post_type=page --post_status=publish --post_title='Verification email' --post_name='verification-email' --post_content='[bis_verify_received_email]'
+wp-env run tests-cli wp post create --post_type=page --post_status=publish --post_title='Notification email' --post_name='notification-email' --post_content='[bis_notification_received_email]'
+
```

Comment:
This didn't seem to work in my testing, as once there are some pages, WC won't create pages like Cart and Checkout and then the e2e tests fail during the step to 
> convert Cart and Checkout pages to shortcode 

Will think about how to address this.